### ISSUE TITLE
Rolling back during music fade out before music statement causes next music to be played on previous statements

### ISSUE THREAD

**Issue by hsandt** [renpy/renpy#4787]
Repro:

```rpy
play music bgm1
"Blabla"
"Let's change music!"
play music bgm2 fadeout 2.0
"New music!"
```

1. Run the game until "Let's change music!"
2. Advance to next statement so current music starts fading out
3. Quickly rollback (mouse wheel up, page up) to previous statement, or even before. Increase fadeout duration if you're not fast enough to give you more time.

Expected: interrupt fade out, or at least wait for it to end, then play previous music
Actual: fade out finishes and game plays the next music despite being in earlier statements

**Comment by renpytom** [comment]
Can you check the nightly fix build? I can't repeat this, and I think there was a relevant fix recently.

**Comment by hsandt** [comment]
Reproduced on nightly fix: 8.1.2.23071001+nightlyupdate
OS: Linux Ubuntu 22.04
